docs/API.md

```diff
@@ -64,3 +64,29 @@ Acceptable formatting:
 
 ### The body of your document
 (required) A longer description of the topic, which will appear on its topic page. Must be 1,000 characters or less. Should not be the same as `short_description`. Some Markdown is allowed, such as links. Emoji are allowed.
+
+##  Making edits to a collection
+
+If you're making edits to a collection, the following fields are available for use. Not all fields are required.
+
+### items
+A comma-separated string containing any of the following values:
+
+- GitHub repository path (e.g. `defunkt/dotjs`)
+- GitHub username (e.g. `defunkt`)
+- GitHub organization (e.g. `github`)
+- Any web URL (e.g. `https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners`)
+- Any YouTube video URL (e.g. `https://www.youtube.com/watch?v=0fKg7e37bQE`)
+
+A complete `items` key-value pair might look like:
+
+`items: pybee/batavia, Homebrew/brew, https://www.youtube.com/watch?v=dSl_qnWO104`
+
+### created_by
+(if applicable) GitHub username of the person and/or organization that authored the collection.
```

Comment:
If we only allow GitHub usernames in this field, an automated test that checks via a regex that the value of `created_by` is a potential username would be good. E.g., `-cats and dogs` is not a valid value.

Reply:
@damaneice Am I correct in assuming that `created_by` expects a GitHub username? Or can it be any value?

Reply:
I will add a test to verify that it matches the correct format. But we aren't linking this to an actual GitHub username in the importer.